archive/ff-history.html

```diff
@@ -446,10 +446,10 @@ <H2>
   svg) fonts.
   <P>
   In June I was thinking of the embolden command I did the year before, and
-  realized that that was essentially the same idea as was needed for generating
+  realized that was essentially the same idea as was needed for generating
   Small Caps glyphs from Capital letters. And then some of those algorithms
   could be used to create condensed and extended glyphs. And then I sat down
-  and wrote a generic "glyph change" dialog -- years ago I had had a "MetaFont"
+  and wrote a generic "glyph change" dialog -- years ago I had a "MetaFont"
```

Comment:
Same as above, it is not non-standard English to repeat 'had'

Reply:
Reminded me of a riddle....
@tshinnic, while @tetedacier had had 'had', had had 'had had'. 'had had' had had a better effect in the file.